powerline/lemonbar.py

```diff
@@ -0,0 +1,23 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+from powerline import Powerline
+from powerline.lib.dict import mergedicts
+
+
+class LemonbarPowerline(Powerline):
```

Comment:
isn't the default implementation okay? I wouldn't know where I should get it from except from the environment locale and thats what `get_preffered_output_encoding` does, no?